all_protos/injective/exchange/v2/tx.proto

```
@@ -378,6 +399,26 @@ message MsgWithdraw {
 // MsgWithdraw defines the Msg/Withdraw response type.
 message MsgWithdrawResponse {}
 
+// MsgUpdateSubaccountRiskProfile defines a SDK message for updating a
+// subaccount's risk profile (e.g. opting into cross-margin).
+message MsgUpdateSubaccountRiskProfile {
+  option (amino.name) = "exchange/MsgUpdateSubaccountRiskProfile";
```

Buf (1.70.0)

[error] cannot find `amino.name` in this scope

(COMPILE)